pkg/edged/containers/container_manager_test.go

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package containers
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/kubeedge/kubeedge/pkg/edged/apis/runtime/cri"
+
+	"github.com/stretchr/testify/assert"
+	"k8s.io/client-go/util/flowcontrol"
+	"k8s.io/kubernetes/pkg/kubelet/gpu"
+	proberesults "k8s.io/kubernetes/pkg/kubelet/prober/results"
+)
+
+// edged socket is the path of the edged registry socket
+var kubePath = "/var/lib/kubelet"
```

Comment:
kubelet -> edged ?

Reply:
This points to a .sock file created and tracked by K8S (vendor) in its code. Edged is not responsible for this file creation. Since NewContainerManager() calls K8S (in vendor), during the process, it is important to check it existance and clean it up; once UT is complete. It helps in development env as it may create un-necessary issues.